modules/rosa-accessing-your-cluster.adoc

```diff
@@ -4,9 +4,14 @@
 
 
 [id="rosa-accessing-your-cluster"]
-= Accessing your cluster
+= Accessing your cluster with an IDP account
 
-To log in to your cluster, you must configure an identity provider (IDP). This procedure uses GitHub as an example IDP. To view other supported IDPs, run `rosa create idp --help`.
+To log in to your cluster, you can configure an identity provider (IDP). This procedure uses GitHub as an example IDP. To view other supported IDPs, run `rosa create idp --help`.
+
+[NOTE]
+====
+Alternatively, as the user who created the cluster, you can use the xref:../getting_started_rosa/accessing-rosa-cluster.html#rosa-accessing-your-cluster-quick[quick access procedure].
```

Comment:
Ideally this `xref` is permissible. But due to reasons, we cannot do this.

If this is essential, you can try to incorporate it into the assembly that includes this module.

Or you can employ some advanced techniques to break up this module, although that'll be somewhat painful. I have some ideas, if you're curious.

Reply:
I removed it. Let's avoid the painful methods, though I appreciate the offer.